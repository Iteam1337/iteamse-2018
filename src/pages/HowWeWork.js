// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/HowWeWorkPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import Header from '../components/Header/Header'
import howWeWorkImage from './img/block_howwework.png'
import howWeWorkFullImage from './img/bleed_howwework.jpg'
import howWeWorkImage2 from './img/block_howwework2.png'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import styled from 'styled-components'

type Props = Iteam.ApolloBase<IteamCMS.HowWeWorkPage>

export const HowWeWorkPageQuery = gql`
  query HowWeWorkPage {
    pageHowWeWork {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      customersText
      customersTitle
      hiringTitle
      methodText
      methodTitle
      sharingText
      sharingTitle
      teamText
      teamTitle
    }
  }
`

const Content = styled.div`
  display: grid;
  grid-row-gap: 40px;
  padding-bottom: 40px;
  padding-top: 40px;

  @media (min-width: 1024px) {
    grid-row-gap: 100px;
    padding-bottom: 100px;
    padding-top: 100px;
  }
`

const HowWeWork = () => {
  return (
    <Query query={HowWeWorkPageQuery}>
      {({ loading, data: { pageHowWeWork } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageHowWeWork.headerImage}
              messageBgColor={pageHowWeWork.headerTextBgColor}
              messageOne={pageHowWeWork.headerText1}
              messageTwo={pageHowWeWork.headerText2}
            />
            <Content>
              <Block title={pageHowWeWork.teamTitle}>
                {pageHowWeWork.teamText}
              </Block>
              <ImageBlock image={howWeWorkImage} />
              <Block title={pageHowWeWork.methodTitle}>
                {pageHowWeWork.methodText}
              </Block>
              <ImageBleed image={howWeWorkFullImage} />
              <Block title={pageHowWeWork.sharingTitle}>
                {pageHowWeWork.sharingText}
              </Block>
              <ImageBlock image={howWeWorkImage2} />
              <Block title={pageHowWeWork.customersTitle}>
                {pageHowWeWork.customersText}
              </Block>
            </Content>
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default HowWeWork
