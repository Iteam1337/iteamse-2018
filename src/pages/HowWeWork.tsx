import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { HowWeWorkPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import { set } from '../utils/googleAnalytics'
import howWeWorkImage2 from './img/illustrations-group-2.svg'
import howWeWorkImage from './img/illustrations-group-3.svg'

export const HOW_WE_WORK_PAGE_QUERY = gql`
  query HowWeWorkPage {
    pageHowWeWork {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      contactTitle
      customersText
      customersTitle
      hiringTitle
      imageBleed
      methodText
      methodTitle
      sharingText
      sharingTitle
      team
      teamText
      teamTitle
    }
  }
`

class HowWeWorkQuery extends Query<HowWeWorkPageQuery> {}

export class HowWeWork extends React.Component {
  componentDidMount() {
    set('hur-vi-jobbar')
  }

  render() {
    return (
      <HowWeWorkQuery query={HOW_WE_WORK_PAGE_QUERY}>
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { pageHowWeWork } = data

          return (
            <>
              <Helmet>
                <title>Iteam - There's a better way |Â Hur vi jobbar</title>
                <meta
                  name="og:title"
                  content="Iteam - There's a better way | Hur vi jobbar"
                />
                <meta
                  name="twitter:title"
                  content="Iteam - There's a better way | Hur vi jobbar"
                />
                {pageHowWeWork.headerImage && (
                  <meta
                    name="og:image"
                    content={`https:${pageHowWeWork.headerImage}`}
                  />
                )}
              </Helmet>
              <Header
                backgroundImage={pageHowWeWork.headerImage}
                messageBgColor={pageHowWeWork.headerTextBgColor}
                messageOne={pageHowWeWork.headerText1}
                messageTwo={pageHowWeWork.headerText2}
              />

              <GridColumn>
                <Block title={pageHowWeWork.teamTitle}>
                  {pageHowWeWork.teamText}
                </Block>
                <ImageBlock image={howWeWorkImage} />
                <Block title={pageHowWeWork.methodTitle}>
                  {pageHowWeWork.methodText}
                </Block>
                <ImageBleed image={pageHowWeWork.imageBleed} />
                <Block title={pageHowWeWork.sharingTitle}>
                  {pageHowWeWork.sharingText}
                </Block>
                <ImageBlock image={howWeWorkImage2} />
                <Block title={pageHowWeWork.customersTitle}>
                  {pageHowWeWork.customersText}
                </Block>
              </GridColumn>

              <Team
                bgColor="blue"
                callToAction={pageHowWeWork.contactTitle}
                shortName={pageHowWeWork.team}
              />
            </>
          )
        }}
      </HowWeWorkQuery>
    )
  }
}

export default HowWeWork
