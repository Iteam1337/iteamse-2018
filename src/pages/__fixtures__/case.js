export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
