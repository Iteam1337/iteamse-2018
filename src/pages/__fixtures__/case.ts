export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
