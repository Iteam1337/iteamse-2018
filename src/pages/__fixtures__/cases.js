export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
