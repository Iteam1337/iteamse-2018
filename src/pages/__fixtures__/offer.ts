// tslint:disable:object-literal-sort-keys
export const offers = [
  {
    offerIntroImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5n7J4mzx3UWam0YwgsUQm/8e00d12d6e80eba927b2a313fb250d87/1.png',
    offerTitle: 'Kod',
    offerIntroText:
      'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med. ',
    offerLead:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. ',
    offerUspOneTitle: 'Utveckling av pilot',
    offerUspOneText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspTwoTitle: 'API',
    offerUspTwoText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspThreeTitle: 'Mobila applikationer',
    offerUspThreeText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspFourTitle: 'AI-lösningar',
    offerUspFourText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi\n',
    offerIllustrationImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5SAhc8YuL6OaQQsC6miokK/7c09d594e44ffa47781bf1bdb21aae7f/illustrations-group-2.png',
    offerOrder: 1,
    __typename: 'Offer',
  },
]
