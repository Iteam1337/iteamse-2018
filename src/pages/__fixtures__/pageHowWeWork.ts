export const pageHowWeWork = {
  __typename: 'PageHowWeWork',
  customersText:
    'Våra kunder har alla stora möjligheter att göra skillnad på sina marknader. De är förbi datorisering och har redan lagt upp alla sina formulär på webben. Nu är de ute efter digitalisering. Och det tar tid. Därför har vi ofta långa relationer med våra kunder. Ofta blir vi vänner längs vägen.',
  customersTitle: 'Långa strömmar, inte korta projekt',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/j8ng4HkakgMCyIMqeek2A/82be3a1d363b21dd29b081e76d0fdfeb/about.jpg',
  headerText1: 'Så här jobbar vi ',
  headerText2: null,
  headerTextBgColor: '#668CFF',
  hiringTitle: 'Just nu söker vi',
  imageBleed:
    '//images.ctfassets.net/rj4r6yfcesw5/62hgzE8G3ueMa2YSEuG4og/db2b643db6041f9d26936bc2d6006607/bleed_howwework.jpg',
  methodText:
    'Vi söker alltid efter den bästa processen. Den bästa metoden. Vi vågar ta risker, byta spår, byta teknik, experimentera och ifrågasätta både oss själva och våra kunder. Det är liksom det vi gör. Hela tiden.',
  methodTitle: 'Ständigt sökande',
  sharingText:
    'På våra kodluncher lär vi av varandra. Vi mentorerar och vi labbar ihop. Vi anordnar Lounge Hack, bidrar till open source, till forumet kodapor tillsammans med 12000 andra utvecklare och deltar på konferenser. Vår kunskap är din kunskap.',
  sharingTitle: 'Delar allt',
  teamText:
    'Vi jobbar alltid i team. Systemutvecklare, strateger, interaktionsdesigners, visuella designers och värdecoacher. Alla är vi värdefulla var och en för sig och ännu mer ihop. Vi jobbar alltid tillsammans. Alltid i tvärfunktionella team. Aldrig uthyrda. Aldrig ensamma.',
  teamTitle: 'I team',
}
