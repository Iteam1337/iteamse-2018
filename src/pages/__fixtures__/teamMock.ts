import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
