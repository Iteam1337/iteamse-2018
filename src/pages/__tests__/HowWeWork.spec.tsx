import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageHowWeWork } from '../__fixtures__/pageHowWeWork'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import HowWeWork, { HOW_WE_WORK_PAGE_QUERY } from '../HowWeWork'

const mocks = [
  {
    request: {
      query: HOW_WE_WORK_PAGE_QUERY,
    },
    result: {
      data: {
        pageHowWeWork,
      },
    },
  },
  TeamQueryMock,
]

describe('components/HowWeWork', () => {
  it('renders HowWeWork', async () => {
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <HowWeWork />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
