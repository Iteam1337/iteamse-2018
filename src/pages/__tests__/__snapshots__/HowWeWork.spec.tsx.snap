// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/HowWeWork renders HowWeWork 1`] = `
<div>
  <div>
    <div
      class="sc-jtRfpW jaOZBl"
    >
      <div
        class="sc-kTUwUJ gOkXip"
      >
        <div
          class="sc-cmTdod cbLNuN"
        >
          <a
            class="sc-jwKygS ccXyBf"
          >
            <img
              class="sc-btzYZH gzqeJw"
              data-testid="logo"
              path="test-file-stub"
            />
          </a>
          <div
            class="sc-lhVmIH gEAOxd"
          >
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Case
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              AI
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="sc-bYSBpT BnQkq"
            />
          </div>
        </div>
        <div
          class="sc-dqBHgY bymaoW"
        >
          <h1
            class="sc-gxMtzJ gChCoz"
          >
            <span>
              Så här jobbar vi 
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              I team
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Vi jobbar alltid i team. Systemutvecklare, strateger, interaktionsdesigners, visuella designers och värdecoacher. Alla är vi värdefulla var och en för sig och ännu mer ihop. Vi jobbar alltid tillsammans. Alltid i tvärfunktionella team. Aldrig uthyrda. Aldrig ensamma.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-VigVT laHvOe"
    >
      <img
        class="sc-jTzLTM jKPziq"
        src="test-file-stub"
      />
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Ständigt sökande
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Vi söker alltid efter den bästa processen. Den bästa metoden. Vi vågar ta risker, byta spår, byta teknik, experimentera och ifrågasätta både oss själva och våra kunder. Det är liksom det vi gör. Hela tiden.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="react-reveal"
    style="opacity: 1;"
  >
    <div
      class="sc-cJSrbW jnWIbK"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-ksYbfQ gohrNG"
        >
          <div
            class="sc-hmzhuo eRtqMp"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Just nu söker vi
            </h3>
            <ul
              class="sc-frDJqD gEgEjh"
            >
              <li
                class="sc-hqyNC jzvVnf"
              >
                <a
                  class="sc-jhAzac eJSshj"
                >
                  Junior UX Designer
                </a>
                (
                Stockholm
                )
              </li>
            </ul>
          </div>
          <div
            class="sc-hmzhuo eRtqMp"
          >
            <h4
              class="sc-kgAjT bykZqW"
            >
              Inget som passar?
            </h4>
            <div
              class="sc-kvZOFW rOLqR"
            >
              <div
                class="sc-htpNat iEyarI"
              >
                <div>
                  <p>
                    <a
                      href="mailto:joinus@iteam.se"
                    >
                      Spontanansök
                    </a>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <img
    class="sc-gqjmRU kaGJTv"
    src="//images.ctfassets.net/rj4r6yfcesw5/62hgzE8G3ueMa2YSEuG4og/db2b643db6041f9d26936bc2d6006607/bleed_howwework.jpg"
  />
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Delar allt
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  På våra kodluncher lär vi av varandra. Vi mentorerar och vi labbar ihop. Vi anordnar Lounge Hack, bidrar till open source, till forumet kodapor tillsammans med 12000 andra utvecklare och deltar på konferenser. Vår kunskap är din kunskap.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-VigVT laHvOe"
    >
      <img
        class="sc-jTzLTM jKPziq"
        src="test-file-stub"
      />
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Långa strömmar, inte korta projekt
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Våra kunder har alla stora möjligheter att göra skillnad på sina marknader. De är förbi datorisering och har redan lagt upp alla sina formulär på webben. Nu är de ute efter digitalisering. Och det tar tid. Därför har vi ofta långa relationer med våra kunder. Ofta blir vi vänner längs vägen.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
